frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -124,7 +128,8 @@ def process_exception(self, request, exception, spider):
     def open(self, spider):
         log.msg('Starting frontier', log.INFO)
         if not self.frontier.manager.auto_start:
-            self.frontier.start()
+            settings_override = getattr(spider, 'frontier_settings', {})
```

Comment:
I also found another interesting issue. Now, we do not expect that someone will be running two or more spiders within the same Scrapy process, however Scrapy allows that http://doc.scrapy.org/en/master/topics/practices.html#running-multiple-spiders-in-the-same-process. Luckily, I don't see any problems with such a setup, but user would need to supply each spider with different storage backend settings.